package.json

```diff
@@ -24,7 +25,7 @@
     "deploy": "gh-pages -d build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "start": "export SET NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
+    "start": "set NODE_OPTIONS=--openssl-legacy-provider && react-scripts start",
```

Comment:
@Vipu97 pls let me know why this was changed 
`export SET` -> `set`
?

Reply:
the react application was not starting with 'npm start' that's why i changed it for running the project in my local machine and it worked after doing so. I will change it back now